SJH/Day7/모의고사.js

```diff
@@ -0,0 +1,31 @@
+function solution(answers) {
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```

해당 문제에서 `answer`는 const를 [사용해도 괜찮아요!](https://github.com/modern-agile-team/11term-assignment/pull/8#discussion_r2602108127)